File: docs/user/en/use/api/example/default-query-example.java
final EvitaResponse<EntityReference> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.query(
			query(
				collection("Product"),
				filterBy(
					and(
						entityPrimaryKeyInSet(1),
						entityLocaleEquals(Locale.ENGLISH),
						priceInPriceLists("basic"),
						priceInCurrency(Currency.getInstance("CZK"))
					)
				)
			),
			EntityReference.class
		);
	}
);

File: documentation/user/en/use/api/example/default-query-example.cs
EvitaResponse<EntityReference> entities = evita.QueryCatalog(
	"evita",
	session => {
		return session.Query<EntityReferenceResponse, EntityReference>(
			Query(
				Collection("Product"),
				FilterBy(
					And(
						EntityPrimaryKeyInSet(1),
						EntityLocaleEquals(new CultureInfo("en")),
						PriceInPriceLists("basic"),
						PriceInCurrency(new Currency("CZK"))
					)
				)
			)
		);
	}
);